changelog/@unreleased/pr-389.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
```

Comment:
probably want to tag this as a `feature` to bump major version

Reply:
I don't think we get a major version bump on features until we release the first major version. Want to call this 1.0.0?

Reply:
I'm inclined to just release as `0.16.0` for now, I think we might want to tackle a couple more (possibly #8 & #278 ) before bumping to `1.0.0`